src/timeMachine/components/SelectorList.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   wrapText?: boolean
 }
 
-const SelectorList: SFC<Props> = props => {
+const SelectorList: FC<Props> = props => {
```

Comment:
Stateless Functional Components are deprecated. Replaced for a Functional component